app/models/customer.py

```diff
@@ -0,0 +1,29 @@
+from app.models import rentals
+from flask import current_app
+from app import db
+from datetime import datetime
+
+class Customer(db.Model):
+    customer_id = db.Column(db.Integer, primary_key=True, autoincrement=True) 
+    name = db.Column(db.String(200))
+    postal_code = db.Column(db.Integer)
+    phone_number = db.Column(db.String(12)) 
+    register_at = db.Column(db.DateTime) 
+    videos_checked_out_count = db.Column(db.Integer, default=0)
+    rental = db.relationship('Rental', backref='rental', lazy=True)
```

Comment:
This should be `rentals` since it's a collection of them:

```suggestion
    rentals = db.relationship('Rental', backref='rental', lazy=True)
```